sagemaker-python-sdk/dgl_gcn/mxnet_gcn_hypertune.ipynb

```diff
@@ -181,13 +181,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now, we'll create a HyperparameterTuner object, which we pass:\n",
+    "Now, create a HyperparameterTuner object, which you pass.\n",
     "\n",
-    " * The training estimator we created above\n",
-    " * Our hyperparameter ranges\n",
+    " * The training estimator you created above\n",
+    " * The hyperparameter ranges\n",
     " * Objective metric name and definition\n",
-    " * Number of training jobs to run in total and how many training jobs should be run simultaneously. More parallel jobs will finish tuning sooner, but may sacrifice accuracy. We recommend you set the parallel jobs value to less than 10% of the total number of training jobs (we'll set it higher just for this example to keep it short).\n",
-    " * Whether we should maximize or minimize our objective metric (we haven't specified here since it defaults to 'Maximize', which is what we want for validation accuracy)"
+    " * Number of training jobs to run in-total and how many training jobs should be run simultaneously. More parallel jobs will finish tuning sooner, but may sacrifice accuracy. We recommend that you set the parallel jobs value to less than 10 percent of the total number of training jobs It's set it higher in this example to keep it short.\n",
+    " * Whether you should maximize or minimize our objective metric. You haven't specified here since it defaults to 'Maximize', which is what you want for validation accuracy)"
```

Comment:
```suggestion
    " * Whether you should maximize or minimize the objective metric. You haven't specified here since it defaults to 'Maximize', which is what you want for validation accuracy"
```